ush/config.sh.RRFS_CONUS_3km_ens

```diff
@@ -165,8 +165,8 @@ if [[ ${DO_ENSEMBLE}  == "TRUE" ]]; then
      DO_RECENTER="TRUE"
 
    fi
-   PPN_RUN_RECENTER="$(( ${NUM_ENS_MEMBERS} + 1 ))"
-   NNODES_RUN_RECENTER="6"
+   PPN_RUN_RECENTER="3"
+   NNODES_RUN_RECENTER="$(( ${NUM_ENS_MEMBERS} + 1 ))"
 #   NNODES_RUN_RECENTER="10"
```

Comment:
The new model code already has everything for making right chunksize restart files. The recenter should use the parallel form.

Reply:
This is what is working for our most recent ensemble experiment so let's keep this change for now. If not needed later, we can remove or change it at a later time.